src/DonationForms/resources/propTypes.ts

```diff
@@ -135,11 +136,13 @@ export interface FormProps {
 
 export interface FieldErrorProps {
     error: string;
+    name: string;
```

Comment:
This is causing some typescript compile errors (`npm run dev`):

<img width="1350" alt="Screenshot 2023-08-31 at 3 12 14 PM" src="https://github.com/impress-org/givewp/assets/10138447/e2eb6f62-929f-417a-b8b6-de947b028047">

Reply:
Resolved in b870e65a4